test/test.js

```diff
@@ -14923,7 +14963,7 @@
     });
 
     test('should throw a TypeError when receiving a non-array `array` argument', 1, function() {
-      raises(function() { _.spread(4, 2); }, TypeError);
```

Comment:
Does `raises` still work? In v3 we avoided `throws` because of some WebKit engines (older for sure by now).

Reply:
When I ran the tests it was throwing errors that `raises` was not defined. Was not aware of the issues with `throws` though.

Reply:
That's okay, no worries. Thank you!